pyproject.toml

```diff
@@ -53,7 +53,7 @@ did-peer-4 = "^0.1.4"
 aries-askar= { version = "~0.3.0", optional = true }
 indy-credx= { version = "~1.1.1", optional = true }
 indy-vdr= { version = "~0.4.0", optional = true }
-anoncreds= { version = "0.2.0.dev11", optional = true, allow-prereleases = true }
+anoncreds= { version = "0.2.0", optional = true, allow-prereleases = true }
```

Comment:
Can (someone correct me if I'm wrong...) I think remove the `allow-prereleases = true` part since 0.2.0 is a release. (I did that when testing plugins in Traction at least and it worked for me)

Reply:
Updated.  I debated doing that when changing and just left, but now removed.